remote-dependency-templates/rds/README.md

```diff
@@ -35,26 +33,25 @@ remote:
         password-secret-name: POSTGRES_PASSWORD
 ```
 
-Environment variables:
-- `DB_NAME`: environment variable for db name, must be the same as the one listed in `<dependency-id>.db-name`
-- `DB_USERNAME` environment variable for db username, must be the same as the one listed in `<dependency-id>.username`
+## Environment variables
+The database name and username should be remote environment variables so that the RDS instance can be accessed. Name them as you choose but ensure they match the values in the template config.
```

Comment:
Maybe: "Name them as you choose but ensure their values match those defined in the template config."